schema/google/showcase/v1beta1/identity.proto

```diff
@@ -41,9 +41,7 @@ service Identity {
       post: "/v1beta1/users"
       body: "*"
     };
-    option (google.api.method_signature) = "user.display_name,user.email";
```

Comment:
These are legitimate method signatures. See https://google.aip.dev/client-libraries/4232#method-signatures_1 : 
* `If a field's name contains a period (.) character, this indicates a nested field.`
* `An RPC can provide this annotation more than once to specify multiple signatures. Order matters here`